DOCUMENTATION.md

```diff
@@ -1467,6 +1505,19 @@ var personTypePrefix = GetNamespacePrefix(
 SetProperty(person, personTypePrefix, "name", "bobby");
 ```
 
+#### AddReference
+
+To set the value of a named reference on an entity use the AddReference function.
+Note that the reference value must be a CURIE.
+Also note that existing references with the same predicate will be overwritten.
+
+```javascript
+var personTypePrefix = AssertNamespacePrefix("http://data.mimiro.io/schema/person/");
+var companyPrefix = AssertNamespacePrefix("http://data.mimiro.io/schema/company/");
+
+AddReference(person, personTypePrefix, "worksfor", companyPrefix+":acme");
```

Comment:
just a small comment: 
might want to use `PrefixField(companyPrefix, "acme")` instead of companyPrefix+":acme" to keep users on builtin functions